modules/security-network-policies.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+//
+// * security/container_security/security-network.adoc
+
+[id="security-network-policies_{context}"]
+= Network policies
+
+Using _network policies_ you isolate pods from each other in the same project.
+Network policies can do such things as deny all network access to a pod,
```

Comment:
Not crazy about "such things", but don't have a better suggestion for wording at the moment.